llmuses/utils/task_utils.py

```diff
@@ -8,7 +8,12 @@ class EvalBackend(Enum):
 
     # Use OpenCompass framework as the evaluation backend
     OPEN_COMPASS = 'OpenCompass'
+    
+    # Use VLM Eval Kit as the multi-modal model evaluation backend
+    VLM_EVAL_KIT = 'VLMEvalKit'
 
     # Use third-party evaluation backend/modules
     THIRD_PARTY = 'ThirdParty'
```

Comment:
i cannot think of useage scenario for such place-holder enum with no specific backend type :)

Reply:
We call that `third party` module such as `llmuses.third_party.toolbench_static`, it's very hard to merge the source code (tool-bench, agent-bench, e.g.) to eval-scope framework, usually we use original implementations to fast integrate these benchmarks.  Otherwise, we can consider another backend name in order to disambiguation.

Reply:
ok. i see.